python.py

```diff
@@ -1 +1,2 @@
-print "Hello World."
\ No newline at end of file
+print "Hello World."
+print "what up people?"
```

Comment:
could have been better to put the message in variable and then print the variable